hudi-common/src/main/avro/HoodieCleanMetadata.avsc

```diff
@@ -24,23 +24,22 @@
      {"name": "totalFilesDeleted", "type": "int"},
      {"name": "earliestCommitToRetain", "type": "string"},
      {"name": "partitionMetadata", "type": {
-     "type" : "map", "values" : {
```

Comment:
this is not deletion. I simply moved this to a separate file for it to be referenced in another place. The idea is to keep these information for bootstrap base files in a separate Avro field for it to be skipped by incremental timeline syncing.